### DISCUSSION TITLE
Remove “/..” from root directory if “restrict_above_cwd” is enabled

### DISCUSSION THREAD

**Discussion by davilera** [nvim-tree/nvim-tree.lua#1735] [Ideas]
Currently, nvim-tree’s default behavior is to append `/..` to the root directory. This makes sense because the user can't browse one directory up. However, if `actions.change_dir.restrict_above_cwd` is enabled, this is no longer possible.

Therefore, I'd recommend appending `/..` in `local function format_root_name` if, and only if, `actions.change_dir.restrict_above_cwd` is **not** enabled. IMO, this would result in a better UX.

**Comment by alex-courtis** [comment]
Added feature request #1743
